client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -13,6 +13,7 @@ import Site from './site';
 import Me from './me';
 import MailingList from './mailing-list';
 import config from 'config';
+import { getQueryCoupon } from 'lib/cart-values';
```

Comment:
This is the import that's causing `client/lib/wp/sync-handler/test/index.js` to fail.